src/main/java/teammates/storage/sqlentity/FeedbackSession.java

```diff
@@ -112,6 +112,35 @@ public FeedbackSession(String name, Course course, String creatorEmail, String i
         this.setPublishedEmailEnabled(isPublishedEmailEnabled);
     }
 
+    /**
+     * Creates a copy that uses the specific deadline for the given user.
+     *
+     * @param userEmail The email address of the given user.
+     * @return The copy of this object for the given user.
+     */
+    public FeedbackSession getCopyForUser(String userEmail) {
+        FeedbackSession copy = getCopy();
+        for (DeadlineExtension de : copy.getDeadlineExtensions()) {
+            if (!de.getUser().getEmail().equals(userEmail)) {
+                de.setEndTime(copy.getEndTime());
+            }
+        }
```

Comment:
There might be a problem here: the deadlineExtensions of this copy is always empty because the deadlineExtensions of the feedback session return by getCopy() is initialized as an empty list